languages/go/config.json

```diff
@@ -61,25 +61,25 @@
       },
       {
         "slug": "constants",
-        "uuid": "31a74993-604d-44ae-afb7-fac7168ccf16",
+        "uuid": null,
         "concepts": ["constants"],
         "prerequisites": ["strings", "numbers"]
       },
       {
         "slug": "context",
-        "uuid": "96aaabad-b8b7-431c-ae59-abb813efd272",
+        "uuid": null,
         "concepts": ["context"],
         "prerequisites": ["time-and-duration", "channels", "select"]
       },
```

Comment:
@tehsphinx I think we should also change the prerequisites here from `time-and-duration ` to `duration`

Reply:
Right... 👍